### DISCUSSION TITLE
Is it possible to support custom entrypoint?

### DISCUSSION THREAD

**Discussion by ngdangtu-vn** [deta/cloud-docs#244] [Ideas] [answered]
I always start my file as `main.*`. The only entrypoint I use index is HTML/PHP. While it is not super important and I can easily rename my file (for small projects) to `index.js`, I still think a custom entrypoint would help developers to port their works form somewhere else or just make us feel comfortable.

**Comment by abdelhai** [comment, marked as answer]
hi, thanks for the input, we're working on reworking custom deployments. for now, having `index.js` is a requirement.
no need to rename your main file. just import the `app` instance from there into `index.js` then reexport.
